src/utils/formHelperFunctions.ts

```diff
@@ -115,7 +115,7 @@ export const lookUpTrigger = (step: any, elementID: any, elementType: any) => {
   let payload = {};
   if (elementType === 'button') {
     const element = step.buttons.find((button: any) => button.id === elementID);
-    payload = { text: element.properties.text };
+    payload = { text: element?.properties?.text ?? '' };
```

Comment:
in what case do we not have the element?

Reply:
This is called from collectPaymentAction in Form/index.tsx.  CollectPaymentAction does a validation first which potentially calls back on the sdk errorCallback which needs a trigger element.  However, this action could be triggered by either a button or a container.  After thinking about it quite a bit, I decided to sort of leave it as is with the trigger looking like a button but with no text nor id property.

Reply:
you should pass it in with elementType = 'container' then. We actually handle that case well since it just falls back to the default payload returned